x-pack/legacy/plugins/maps/public/layers/styles/vector/components/legend/_category.scss

```
@@ -0,0 +1,3 @@
+.iconPreview {
```

This className is too generic. Just to avoid possible conflicts it should be something like:
```suggestion
.mapLegendIconPreview {
```